### ISSUE TITLE
tfjs classification model in react native app - Error: Unknown layer: Normalization

### ISSUE THREAD

**Issue by brianspiesman** [tensorflow/tfjs#7639]
I trained a custom EfficientNetB2 classification model with Tensorflow 2.9.0: 
```
base_model = EfficientNetB2(include_top=False, pooling ='avg', weights='imagenet', input_shape=(IMG_SIZE, IMG_SIZE, 3))
x = base_model.output
x = Dense(1024, activation='relu')(x)
x = Dropout(0.01)(x)
predictions = Dense(num_classes, activation='softmax')(x)
model = Model(inputs = base_model.input, outputs = predictions)
```

I then converted the resulting .h5 saved model to tfjs (using tensorflowjs 3.19.0):
`tfjs.converters.save_keras_model(saved_model, 'tfjs_model') `

After the conversion completes I get this message:
`weight normalization_3/count with shape () and dtype int64 was auto converted to the type int32
`

**Then, when I use the tfjs model in a react native app, I get this error message:** 
```
Possible Unhandled Promise Rejection (id: 2):
Error: Unknown layer: Normalization. This may be due to one of the following reasons:
1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.
2. The custom layer is defined in JavaScript, but is not registered properly with tf.serialization.registerClass().
```
I am using @tensorflow/tfjs 4.4.0

This seems to be an error related to normalization but I am not sure how to address it.

Any help would be appreciated!

**Comment by brianspiesman** [comment]
@Linchenn any ideas here?

**Comment by geminigeek** [comment]
having same issue !! i trained a model in  EfficientNetB0 , same error is showing , did you find any solution ?

**Comment by brianspiesman** [comment]
Nope. No response. I've moved on to a different approach.

**Comment by fernandozet** [comment]
I used tensorflow.keras.layers.Normalization() in my model and I am getting the same error. Did you guys find a workaround for this issue?

**Comment by alena424** [comment]
same issue

**Comment by jatinvinkumar** [comment]
I have the same issue. Has anyone found a workaround?

**Comment by KeningTown** [comment]
got same issu, maybe someone know what to do?

**Comment by vmukhachev** [comment]
@KeningTown , IT MAY BE the workaroud is to write custom Normalization layer and register it using tf.serialization.registerClass

**Comment by nileshau-afk** [comment]
Hello @brianspiesman  ,

Thank you for bringing this to our attention and for providing a comprehensive overview. We appreciate the time you invested in creating this issue.

Currently, our internal development team is concentrated on other scheduled milestones, so we cannot prioritize this task immediately. However, we would love to see this move forward as a community-led effort.

If you’d like to submit a fix, please fork the repo and open a PR following the [TensorFlow Contribution Guidelines](https://www.tensorflow.org/community/contribute), [Tfjs Contributions](https://github.com/tensorflow/tfjs/blob/master/tfjs-vis/CONTRIBUTING.md#how-to-contribute) and be sure to link this issue in your PR to avoid duplicate work, kindly leave a comment here if you plan to take this on.

Thank you again for your interest and time.

**Comment by Varun-S10** [comment]
Hi @brianspiesman, Thank you for your patience, we have reproduced the issue. As a workaround, you can register a custom Normalization layer class in JavaScript extending TensorFlow.js's Layer class and register it using the serialization registry before loading your model with loadLayersModel. Alternatively, you can convert your model to the TensorFlow.js Graph Model format using the converter tool and load it via loadGraphModel, or perform feature normalization on your input tensors prior to calling predict. Could you please try this and let us know if it works for you?
